apps/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ if (BUILD_GUI)
 endif()
 
 if (BUILD_PYTHON)
-    add_subdirectory (vapi)
+    add_subdirectory (pythonapi)
```

Comment:
Can you comment on the difference between "vapi" and "pythonapi"?  Is there a difference?

Reply:
`vapi` is the "vapor api" aka a C++ library from which you can use Vapor.
`pythonapi` is the Python API.